clusterloader2/testing/l4ilb/config.yaml

```diff
@@ -12,6 +12,9 @@
 {{$ilbWaitTimeout := DefaultParam .CL2_ILB_WAIT_TIMEOUT "10m"}}
 {{$namespaces := 1}}
 
+# Command to be executed
```

Comment:
Quite a lot of behaviour is changed when setting this single flag, with very generic name.

Wouldn't it be better to have separate flags controlling initial ILB testing and node pool upgrade? Does it make sense to run both tests serially?

Reply:
Can we extract stuff into modules and instead of "if-ing" in a bunch of places create two separate tests from the common modules?

Reply:
I think it would make sense to run them both serially. This way the existing test would remain the same if Exec command override is not provided, and if it is, then both are run.
It would work when removing all other {{if}}s.

Does that seem reasonable?

Reply:
Sounds good to me.